dbt/adapters/athena/impl.py

```diff
@@ -57,6 +57,73 @@ def convert_number_type(cls, agate_table: agate.Table, col_idx: int) -> str:
     def convert_datetime_type(cls, agate_table: agate.Table, col_idx: int) -> str:
         return "timestamp"
 
+    def _parse_lf_tags(self, lf_tags_expr: Optional[str]) -> Dict[str, str]:
+        lf_tags = {}
+        if lf_tags_expr:
+            lf_tags = dict([tag.split("=") for tag in lf_tags_expr.split(",")])
+
+        return lf_tags
+
+    @available
+    def add_lf_tags_to_table(self, database: str, table: str, lf_tags_expr: Optional[str] = None):
```

Comment:
this function and add_lf_tags_to_database are really similar, excep the resouce type, why don't you refactor a bit?

you have 2 options:
1. you create a generic `add_lf_tags(destination, tags...)` where destination is an object that can be passed based on the context, in case of database will have an input in case of tables will have another input build directly from jinjia.
2. similar to 1, you create a generic `add_lf_tags` that is then invoked by add_lf_tags_to_database and add_lf_tags_to_table

The idea is to avoid repetition, and I think that in this case we can :)

Reply:
@nicor88 totally agree, I made the updates requested